src/util/index.ts

```diff
@@ -121,3 +121,26 @@ export const getIconGlyphClasses = (name: IconGlyphs) => {
 export const formatFloatForDisplay = (float: number): string => {
     return parseFloat(float.toFixed(2)).toString();
 };
+
+export const copyToClipboard = async (text: string): Promise<void> => {
+    try {
+        await navigator.clipboard.writeText(text);
+    } catch (err) {
+        console.error("Failed to copy text: ", err);
+    }
+};
+
+export const getUrlParamValue = (url: string, param: string) => {
+    const urlObj = new URL(url);
+    const params = new URLSearchParams(urlObj.search);
+    return params.get(param);
+};
```

Comment:
nit, because there are other param functions in this file, so maybe for another PR, but this should probably be in urlHandling instead

Reply:
https://github.com/simularium/simularium-website/pull/551